gramps_webapi/api/resources/base.py

```diff
@@ -65,22 +69,24 @@ def get(self, args: Dict, handle: str) -> Response:
         try:
```

Comment:
This method is getting really long; it would be more readable by putting the various `if` cases into separate methods. But I leave it up to you.

Reply:
This will need to be refactored for sorting so can address it then.  I think I agree with you, handle goes away. With the HasIdOf rule the gramps_id is not needed either.